src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=" "
+      method="post"
+      class="search big"
+      data-qa="big"
+    >
+      <button type="submit" class="button-big">
```

Comment:
I think search icon isn't submit, and after click I got
<img width="480" alt="Screenshot 2023-03-08 at 10 48 40" src="https://user-images.githubusercontent.com/53228259/223666319-7d24f20f-a822-4443-8a32-97abd73200e5.png">